pygenometracks/tracks/EpilogosTrack.py

```diff
@@ -90,6 +90,10 @@ def plot(self, ax, chrom_region, start_region, end_region):
             # the qcat_json is a pseudo json line, that misses
             # the { } and the quotes. The following lines fix that.
             qcat_json = qcat_json[0]
+            if not isinstance(qcat_json, str):
+                # This would happen if the qcat file has a hole.
```

Comment:
What about:

```suggestion
                # This would happen if the qcat file has a missing value.
                #The missing value is filled with np.repeat(np.nan, ..) and should be skipped here.
```